### ISSUE TITLE
Application has no Accessibility features

### ISSUE THREAD

**Issue by noquierouser** [telegramdesktop/tdesktop#1891]
### Steps to reproduce
1. Open application with JAWS

### Expected behaviour
Application's content should be readable by JAWS and it should read the application's content to the user.

### Actual behaviour
No application content is read, only textboxes are described.

### Configuration
**Operating system:**
- Windows 10
- JAWS 17

**Version of Telegram Desktop:**
0.9.42

I was just testing if the app is accessible and what is the experience if I tried to use it with a screen reader, but JAWS wasn't able to describe the app at all, not even a titlebar description.

Is there any consideration for accessibility features in the application in the future?

**Comment by ivmirx** [comment]
Same for OS X. The app's UIs are not recognized by system accessibility tools.

**Comment by auchri** [comment]
Duplicate of #476
